tools/generate_typing_context.py

```diff
@@ -84,7 +85,9 @@ def generate_protocols(file, class_name):
         if ast.get_docstring(call_function):
             call_function.body = call_function.body[:1]
         else:
-            call_function.body = [ast.Expr(ast.Constant(...))]
+            call_function.body = []
+
+        call_function.body.append(ast.Expr(ast.Constant(...)))
```

Comment:
ATL: why do we need this?

Reply:
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/2745502/183962112-0de160de-a12c-44dc-b384-e277713c020e.png">

Reply:
Seems like a bug in pylance TBH (protocols are not supposed return a thing). But if this makes it work, perfect 💯